### PR TITLE
Add Transcript APIs and Resources.

### PR DESCRIPTION
**Pull request by mankadnandan**

### CODE REVIEW

File: api-reference/beta/api/callTranscript-get-content.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+title: "Get callTranscript content"
+description: "Retrieve single callTranscript content associated to a Microsoft Teams Online Meeting."
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:3]: associated with* ?

File: api-reference/beta/api/callTranscript-get-content.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+HTTP/1.1 200 OK
+Content-type: text/vtt
+
+WEBVTT
+
+0:0:0.0 --> 0:0:5.320
+<v User Name>This is a transcript test.</v>
+```
+
+### Example 3: Get a callTranscript Content specifying Accept query param
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:118]: Accept header

File: api-reference/beta/api/callTranscript-get-content.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+  "name": "get_callTranscript_content_Accept"
+}
+-->
+``` http
+GET https://graph.microsoft.com/beta/users('ba321e0d-79ee-478d-8e28-85a19507f456')/onlineMeetings('MSo1N2Y5ZGFjYy03MWJmLTQ3NDMtYjQxMy01M2EdFGkdRWHJlQ')/transcripts('MSMjMCMjNzU3ODc2ZDYtOTcwMi00MDhkLWFkNDItOTE2ZDNmZjkwZGY4')/content
+Accept: application/vnd.openxmlformats-officedocument.wordprocessingml.document
+```
+#### Response
+
+Response contains bytes for the transcript in the body. `content-type` header specifies type of the transcript content.
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:131]: Uppercase c in Content-Type

File: api-reference/beta/api/callTranscript-get-content.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+}
+-->
+```http
+HTTP/1.1 200 OK
+Content-type: application/vnd.openxmlformats-officedocument.wordprocessingml.document
+
+<DOCX Content>
+```
+
+### Example 4: Get a callTranscript Content with $format getting precedence
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:146]: maybe complete the sentence- getting precedence over the Accept header

File: api-reference/beta/api/callTranscript-get.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
+---
+title: "Get callTranscript"
+description: "Retrieve single callTranscript associated to a Microsoft Teams Online Meeting."
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get.md:3]: associated with*

Comment (shantanu1singh) [reply api-reference/beta/api/callTranscript-get.md:3]: here and in other places

File: api-reference/beta/api/callTranscript-get.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | OnlineMeetingArtifact.Read.All |
+|Delegated (personal Microsoft account) | Not supported.    |
+|Application | Not supported. |
+
+## HTTP request
+
+<!-- { "blockType": "ignored" } -->
+```http
+users({userId})/onlineMeetings({meetingId})/transcripts({transcriptId})
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get.md:32]: Prefix with GET

File: api-reference/beta/api/callTranscript-get-content.md [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | OnlineMeetingArtifact.Read.All |
+|Delegated (personal Microsoft account) | Not supported.    |
+|Application | Not supported. |
+
+## HTTP request
+
+<!-- { "blockType": "ignored" } -->
+```http
+users({userId})/onlineMeetings({meetingId})/transcripts({transcriptId})/content
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:32]: Prefix with GET

Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:32]: We should also include an example with /me.
That's what they seem to be doing for attendance report too: https://docs.microsoft.com/en-us/graph/api/meetingattendancereport-get?view=graph-rest-1.0

File: api-reference/beta/api/callTranscript-get.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+```
+
+## Optional query parameters
+
+This operation does not currently support [OData query parameters](/graph/query-parameters) to customize the response.
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get.md:37]: Can we remove this section entirely if it's not supported?

File: api-reference/beta/api/callTranscript-list.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
+﻿---
+title: "List callTranscripts"
+description: "Retrieve the list of transcripts associated to a Microsoft Teams Online Meeting."
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-list.md:3]: associated with*. Here and in other places

File: api-reference/beta/toc.yml [hunk 10]
```diff
@@ -14844,6 +14844,18 @@ items:
           href: api/onenote-list-notebooks.md
         - name: Create notebook
           href: api/onenote-post-notebooks.md
+      - name: Online meetings
```
Comment (shantanu1singh) [inline api-reference/beta/toc.yml:14847]: I can see an existing section for Online meetings in the toc. Let's update that one instead to include the new docs

File: api-reference/beta/api/callTranscript-get-content.md (hunk 1 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get-content.md]: Done

File: api-reference/beta/api/callTranscript-get-content.md (hunk 2 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get-content.md]: Done

File: api-reference/beta/api/callTranscript-get-content.md (hunk 3 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get-content.md]: Done

File: api-reference/beta/api/callTranscript-get.md (hunk 6 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get.md]: Done

File: api-reference/beta/api/callTranscript-get.md (hunk 5 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get.md]: Done

File: api-reference/beta/api/callTranscript-get-content.md (hunk 7 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get-content.md]: Done

Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get-content.md]: Added

File: api-reference/beta/api/callTranscript-list.md (hunk 9 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-list.md]: Done

File: api-reference/beta/api/callTranscript-get.md (hunk 8 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get.md]: Done

File: api-reference/beta/toc.yml (hunk 10 above)
Comment (mankadnandan) [reply api-reference/beta/toc.yml:14847]: The existing section for Online meetings falls under Cloud Communications. I thought of adding this under Teamwork for transcripts.

File: api-reference/beta/api/callTranscript-get-content.md [hunk 11]
```diff
@@ -25,6 +25,9 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | Not supported. |
 
+> [!NOTE]
+> This API is in beta and might undergo changes with respect to the request and/or response schema. Use of this API in production applications is not recommended.
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:29]: Maybe just say This API might undergo changes...

since they already know that this is the /beta API documentation overall

**Review by shantanu1singh — APPROVED** [review]
LGTM

**Comment by Lauragra** [comment]
@mankadnandan , is this PR ready for content review? If so, please apply the **ready for content review** label. The schema associated with the PR has already been release so we need to get the docs published. Thanks.

File: api-reference/beta/api/callTranscript-list.md [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+        },
+        {
+            "id": "MSMjMCMjZjljOTJkMjgtODZiNS00OTQ5LWE3ZDUtZDZhMDNiNWI4ZGNm",
+            "createdDateTime": "2021-09-16T14:00:47.960732Z"
+        },
+        {
+            "id": "MSMjMCMjYWQ4ZDAzNWMtNDk2Yi00MGRmLTk3ZmUtNmVjMTgyYzFkYWM0",
+            "createdDateTime": "2021-09-16T13:59:27.1210383Z"
+        }
+    ]
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-list.md:132]: Not sure but maybe reduce the number of items we are returning in the sample response just to keep things concise.

File: api-reference/beta/api/callTranscript-list.md [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+
+## Optional query parameters
+
+This method supports the `$skipToken` and `$top` [OData query parameters](/graph/query-parameters) to help customize the response.
+
+### Supported query patterns
+
+| Pattern                | Supported | Syntax                                 | Notes |
+| ---------------------- | ------- | -------------------------------------- | ----- |
+| Server-side pagination |     ✓     | `@odata.nextLink`                      | |
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-list.md:46]: Looks like the following was added under the 'Notes' column for servcer-side pagination in other docs:

You will get a continuation token in the response, when a result set spans multiple pages.

File: api-reference/beta/api/callTranscript-get-content.md [hunk 14]
```diff
@@ -0,0 +1,183 @@
...
+
+<!-- { "blockType": "ignored" } -->
+```http
+GET me/onlineMeetings({meetingId})/transcripts({transcriptId})/content
+GET users({userId})/onlineMeetings({meetingId})/transcripts({transcriptId})/content
+```
+
+## Optional query parameters
+
+This method supports the `$format` [OData query parameters](/graph/query-parameters) to help customize the response.
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:43]: singular - parameter?

Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:43]: Maybe reword to the following?

This method supports the `$format` [OData query parameter](/graph/query-parameters) that allows response customization?

File: api-reference/beta/api/callTranscript-get-content.md [hunk 15]
```diff
@@ -0,0 +1,183 @@
...
+
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and contains bytes for [callTranscript](../resources/callTranscript.md) object in the response body. `content-type` header specifies type of the transcript content.
+
+## Examples
+
+### Example 1: Get a callTranscript Content
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:64]: lowercase - content?

File: api-reference/beta/api/callTranscript-get-content.md [hunk 16]
```diff
@@ -0,0 +1,183 @@
...
+HTTP/1.1 200 OK
+Content-type: text/vtt
+
+WEBVTT
+
+0:0:0.0 --> 0:0:5.320
+<v User Name>This is a transcript test.</v>
+```
+
+### Example 2: Get a callTranscript Content specifying $format query param
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:94]: lowercase - content?

File: api-reference/beta/api/callTranscript-get-content.md [hunk 17]
```diff
@@ -0,0 +1,183 @@
...
+HTTP/1.1 200 OK
+Content-type: text/vtt
+
+WEBVTT
+
+0:0:0.0 --> 0:0:5.320
+<v User Name>This is a transcript test.</v>
+```
+
+### Example 3: Get a callTranscript Content specifying Accept header
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:124]: lowercase - content?

File: api-reference/beta/api/callTranscript-get-content.md [hunk 18]
```diff
@@ -0,0 +1,183 @@
...
+```http
+HTTP/1.1 200 OK
+Content-type: application/vnd.openxmlformats-officedocument.wordprocessingml.document
+
+0:0:0.0 --> 0:0:5.320
+User Name
+This is a transcript test.
+```
+
+### Example 4: Get a callTranscript Content with $format getting precedence over the Accept header
```
Comment (shantanu1singh) [inline api-reference/beta/api/callTranscript-get-content.md:154]: lowercase - content?

**Review by shantanu1singh — APPROVED** [review]

File: api-reference/beta/api/callTranscript-list.md (hunk 13 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-list.md]: Added

File: api-reference/beta/api/callTranscript-get-content.md (hunk 14 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get-content.md]: Done

File: api-reference/beta/api/callTranscript-get-content.md (hunk 15 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get-content.md]: Done

File: api-reference/beta/api/callTranscript-get-content.md (hunk 16 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get-content.md]: Done

File: api-reference/beta/api/callTranscript-get-content.md (hunk 17 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get-content.md]: Done

File: api-reference/beta/api/callTranscript-get-content.md (hunk 18 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get-content.md]: Done

File: api-reference/beta/resources/callTranscript.md [hunk 19]
```diff
@@ -0,0 +1,47 @@
+---
```
Comment (isvargasmsft) [inline api-reference/beta/resources/callTranscript.md:1]: Please remove the capitalization from the file name. It should be calltranscript.md

Comment (isvargasmsft) [reply api-reference/beta/resources/callTranscript.md:1]: Please apply to the other file names. We use camel caps for titles but small caps for file names.

File: api-reference/beta/resources/callTranscript.md [hunk 20]
```diff
@@ -0,0 +1,47 @@
...
+|[List callTranscripts](../api/callTranscript-list.md) | [callTranscript](callTranscript.md) collection | Get the list of transcripts associated to an Online Meeting.| 
+|[Get callTranscript](../api/callTranscript-get.md) | [callTranscript](callTranscript.md) | Read properties of a transcript.| 
+|[Get callTranscript content](../api/callTranscript-get-content.md) | [callTranscript](callTranscript.md) content bytes | Get the transcript's content bytes.| 
+
+## Properties
+
+| Property   | Type |Description|
+|:---------------|:--------|:----------|
+| id| String| The transcript's unique identifier. Read-only.|
+| createdDateTime| dateTimeOffset|  Date and time at which the transcript was created. Read-only.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/callTranscript.md:30]: Property missing: content.
 <Property Name="content" Type="Edm.Stream" />

File: api-reference/beta/resources/callTranscript.md [hunk 21]
```diff
@@ -0,0 +1,47 @@
...
+<!-- {
+  "blockType": "resource",
+  "keyProperty": "id",
+  "@odata.type": "microsoft.graph.callTranscript"
+}-->
+
+```json
+{
+  "id": "string (identifier)",
+  "createdDateTime": "dateTimeOffset",
```
Comment (isvargasmsft) [inline api-reference/beta/resources/callTranscript.md:45]: Missing property: content.
 <Property Name="content" Type="Edm.Stream" />

File: api-reference/beta/api/callTranscript-get-content.md [hunk 22]
```diff
@@ -0,0 +1,183 @@
+---
+title: "Get callTranscript content"
+description: "Retrieve single callTranscript content associated with a Microsoft Teams Online Meeting."
+author: "mankadnandan"
+ms.localizationpriority: medium
+ms.prod: "microsoft-teams"
+doc_type: apiPageType
+---
+
+# Get callTranscript content
```
Comment (isvargasmsft) [inline api-reference/beta/api/callTranscript-get-content.md:10]: It seems like the content property is being treated as a navigation property, but in the CSDL it is being introduced as a regular property. Please clarify.

File: api-reference/beta/api/callTranscript-list.md [hunk 23]
```diff
@@ -0,0 +1,106 @@
+﻿---
```
Comment (isvargasmsft) [inline api-reference/beta/api/callTranscript-list.md:1]: I believe the file name should be: onlinemeeting-list-transcripts. According to the file name guidelines: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211751/File-naming?anchor=microsoft-graph-file-naming-guidelines

image

File: api-reference/beta/resources/onlinemeeting.md [hunk 24]
```diff
@@ -95,6 +95,7 @@ This resource supports subscribing to [change notifications](/graph/webhooks).
 | attendanceReports | [meetingAttendanceReport](meetingAttendanceReport.md)  collection | The attendance reports of an online meeting. Read-only. |
 | registration | [meetingRegistrationBase](meetingregistrationbase.md) | The registration that has been enabled for an online meeting. One online meeting can only have one registration enabled.|
 | meetingAttendanceReport (deprecated) | [meetingAttendanceReport](meetingAttendanceReport.md) | The attendance report of the latest online meeting session. Read-only. |
+| transcripts | [callTranscript](callTranscript.md) collection | The transcripts of an online meeting. Read-only. |
```
Comment (isvargasmsft) [inline api-reference/beta/resources/onlinemeeting.md:98]: In the methods column in this topic, please add the List transcripts operation. We should also add the List operations for the attendanceReport navigation property that is also a collection of objects.

File: changelog/Microsoft.Teams.Core.json [hunk 25]
```diff
@@ -1,5 +1,49 @@
...
+          "Target": "callTranscript"
+        },
+        {
+          "Id": "09e85d0c-2551-40e7-a037-1f36fade0b0b",
+          "ApiChange": "Method",
+          "ChangedApiName": "List transcripts, Get transcript, Get transcript content",
+          "ChangeType": "Addition",
+          "Description": "Added the [List Transcripts](https://docs.microsoft.com/en-us/graph/api/callTranscript-list?view=graph-rest-beta), [Get Transcript](https://docs.microsoft.com/en-us/graph/api/callTranscript-get?view=graph-rest-beta), [Get Transcript Content] (https://docs.microsoft.com/en-us/graph/api/callTranscript-get-content?view=graph-rest-beta) methods.",
+          "Target": "callTranscript"
+        }
```
Comment (isvargasmsft) [inline changelog/Microsoft.Teams.Core.json:20]: Please follow the guidelines to add changelog entries for CRUD operations, specifically example 6 of this list: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog?anchor=changelist-record-properties

We should add one changelog entry for each element.

File: changelog/Microsoft.Teams.Core.json [hunk 26]
```diff
@@ -1,5 +1,49 @@
...
+      "Version": "beta",
+      "CreatedDateTime": "2021-12-27T17:37:11Z",
+      "WorkloadArea": "Teamwork",
+      "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "ac7992ce-3a29-456a-88b9-934313fdd0ea",
+          "ApiChange": "Resource",
```
Comment (isvargasmsft) [inline changelog/Microsoft.Teams.Core.json:33]: In this case, the ApiChange is not a resource, but a Relationship. Please follow example 3 here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog?anchor=changelist-record-examples

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @mankadnandan, 

Thank you very much for your work in this PR. 

Just a few comments:
1. Please remove the capitalization from file names and review the structure of file name for list operations. 
2. Please clarify of the content property of callTranscript is a navigation property or a regular property. And please document it accordingly in the callTranscript resource topic. 
3. Please take a look at some changelog documentation issues. I pasted the guidelines with examples in the comments. 

Regarding the TOC and the area, please let me ask whether this belongs to teamwork or to cloud communications. 

Please ping me on the side if you have questions. 

Kind regards, 

Isaac

File: api-reference/beta/resources/callTranscript.md (hunk 19 above)
Comment (mankadnandan) [reply api-reference/beta/resources/callTranscript.md]: Thanks. Done.

File: api-reference/beta/resources/callTranscript.md (hunk 20 above)
Comment (mankadnandan) [reply api-reference/beta/resources/callTranscript.md]: The content property is not directly exposed in the response.

File: api-reference/beta/resources/callTranscript.md (hunk 21 above)
Comment (mankadnandan) [reply api-reference/beta/resources/callTranscript.md]: The content property is not directly exposed in the response.

File: api-reference/beta/api/callTranscript-get-content.md (hunk 22 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-get-content.md]: Content property is a regular property but is not directly exposed in the response.

File: api-reference/beta/api/callTranscript-list.md (hunk 23 above)
Comment (mankadnandan) [reply api-reference/beta/api/callTranscript-list.md]: I see. I thought it was more intuitive to have the file named in the current way as the get API are also named in the similar fashion. Let me know if thats fine or we should follow the guideline and I can rename it accordingly.

File: api-reference/beta/resources/onlinemeeting.md (hunk 24 above)
Comment (mankadnandan) [reply api-reference/beta/resources/onlinemeeting.md:98]: Sure. Should we add only the list method or we should also add Get and Get Content methods too? Actually attendanceReport is not owned by our team so we might have to involve Frank Peng for it.

File: changelog/Microsoft.Teams.Core.json (hunk 25 above)
Comment (mankadnandan) [reply changelog/Microsoft.Teams.Core.json]: Sure, changed.

File: changelog/Microsoft.Teams.Core.json (hunk 26 above)
Comment (mankadnandan) [reply changelog/Microsoft.Teams.Core.json]: Sure, changed.

**Comment by mankadnandan** [comment]
> 
Hi @isvargasmsft Thanks for your review. Appreciate it.

I have made most of the changes. I have replied for few of them where I had a few questions. Let me know and I will work on them quickly to push them.

**Comment by mankadnandan** [comment]
@Lauragra - While @isvargasmsft is out this week, can we get someone to check this PR?

File: api-reference/beta/resources/onlinemeeting.md (hunk 24 above)
Comment (isvargasmsft) [reply api-reference/beta/resources/onlinemeeting.md:98]: Only the list transcript method, please.

File: api-reference/beta/toc.yml (hunk 10 above)
Comment (isvargasmsft) [reply api-reference/beta/toc.yml:15262]: Checking with the docs team. Please bear with me.

**Comment by isvargasmsft** [comment]
Blocking this PR until we resolve whether these resources are for private preview or beta. 

After checking with architects, we found some inconsistencies in the schema files. 

Please review with onboarding team and clarify.

**Comment by mankadnandan** [comment]
> Blocking this PR until we resolve whether these resources are for private preview or beta.
> 
> After checking with architects, we found some inconsistencies in the schema files.
> 
> Please review with onboarding team and clarify.

Hi @isvargasmsft  - This

File: api-reference/beta/resources/calltranscript.md [hunk 27]
```diff
@@ -0,0 +1,47 @@
...
+|[List callTranscripts](../api/calltranscript-list.md) | [callTranscript](callTranscript.md) collection | Get the list of transcripts associated to an Online Meeting.| 
+|[Get callTranscript](../api/calltranscript-get.md) | [callTranscript](callTranscript.md) | Read properties of a transcript.| 
+|[Get callTranscript content](../api/calltranscript-get-content.md) | [callTranscript](callTranscript.md) content bytes | Get the transcript's content bytes.| 
+
+## Properties
+
+| Property   | Type |Description|
+|:---------------|:--------|:----------|
+| id| String| The transcript's unique identifier. Read-only.|
+| createdDateTime| dateTimeOffset|  Date and time at which the transcript was created. Read-only.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/calltranscript.md:30]: Please add the property "content":
Even if it is not returned in get operations, it is still a valid property and should be documented.

File: api-reference/beta/resources/calltranscript.md [hunk 28]
```diff
@@ -0,0 +1,47 @@
...
+<!-- {
+  "blockType": "resource",
+  "keyProperty": "id",
+  "@odata.type": "microsoft.graph.callTranscript"
+}-->
+
+```json
+{
+  "id": "string (identifier)",
+  "createdDateTime": "dateTimeOffset",
```
Comment (isvargasmsft) [inline api-reference/beta/resources/calltranscript.md:45]: Please add the property "content" as per my last comment.

File: api-reference/beta/toc.yml [hunk 29]
```diff
@@ -16153,6 +16153,18 @@ items:
...
```
Comment (isvargasmsft) [inline api-reference/beta/toc.yml:16156]: Please use the existing Online meetings node in the TOC. Do not create additional node under Teamwork for now.

File: api-reference/beta/resources/onlinemeeting.md [hunk 30]
```diff
@@ -95,6 +95,7 @@ This resource supports subscribing to [change notifications](/graph/webhooks). S
...
```
Comment (isvargasmsft) [inline api-reference/beta/resources/onlinemeeting.md:98]: The CSDL PR is not introducing this new navigation property to beta, only the new entity. Could you please clarify? Is there another CSDL PR?

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello team, 

Thank you very much for the work in this PR. Please review the following:
1. Let's use the existing Online meetings node in the TOC. Do not create a new one under Teamwork for now. 
2. Please add the property "content" to the properties table and the JSON representation of the resource topic. Even if it is not returned by GET operations, it is still a valid property and should be documented.
3. I noticed the CSDL PR is not introducing the new navigation property (callTranscripts) to the onlineMeeting resource. Please take a look. If there is another CSDL PR that introduced the change, please share it with me on the side. 

Thank you very much. 

Isaac

File: api-reference/beta/resources/calltranscript.md (hunk 27 above)
Comment (mankadnandan) [reply api-reference/beta/resources/calltranscript.md:30]: Sure, makes sense. Added.

File: api-reference/beta/resources/calltranscript.md (hunk 28 above)
Comment (mankadnandan) [reply api-reference/beta/resources/calltranscript.md:46]: Added.

File: api-reference/beta/toc.yml (hunk 29 above)
Comment (mankadnandan) [reply api-reference/beta/toc.yml]: Sure, thanks. Done.

File: api-reference/beta/resources/onlinemeeting.md (hunk 30 above)
Comment (mankadnandan) [reply api-reference/beta/resources/onlinemeeting.md:98]: In the original CSDL PR https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5458466?_a=files&path=/Schemas/beta-Prod.csdl we have added that on the line number 23255.

Let me know if this looks good.

File: api-reference/beta/api/calltranscript-get.md [hunk 31]
```diff
@@ -0,0 +1,87 @@
...
+
+> [!NOTE]
+> This API would work for a meeting only if the meeting has not expired. For more details, see [Limits and specifications for Microsoft Teams](/microsoftteams/limits-specifications-teams#meeting-expiration).
+
+## HTTP request
+
+<!-- { "blockType": "ignored" } -->
+```http
+GET me/onlineMeetings({meetingId})/transcripts({transcriptId})
+GET users({userId})/onlineMeetings({meetingId})/transcripts({transcriptId})
```
Comment (isvargasmsft) [inline api-reference/beta/api/calltranscript-get.md:38]: Please include the leading /
```suggestion
GET /me/onlineMeetings({meetingId})/transcripts({transcriptId})
GET /users({userId})/onlineMeetings({meetingId})/transcripts({transcriptId})
```

File: api-reference/beta/api/calltranscript-get-content.md [hunk 32]
```diff
@@ -0,0 +1,186 @@
+---
+title: "Get callTranscript content"
+description: "Retrieve single callTranscript content associated with a Microsoft Teams Online Meeting."
+author: "mankadnandan"
+ms.localizationpriority: medium
+ms.prod: "microsoft-teams"
+doc_type: apiPageType
+---
+
+# Get callTranscript content
```
Comment (isvargasmsft) [inline api-reference/beta/api/calltranscript-get-content.md:10]: After consulting with my team, we decided the consistent way to show this use case of getting the content of a transcript is in the calltranscript-get.md topic. Make a mention of that in the top section; link to an example below. In the HTTP request section, show a separate description and syntax line after the existing 2 lines (lines 36-39). In the Examples section, add an example with a section title that captures getting the content of a transcript.

We can then delete this separate topic.

File: changelog/Microsoft.Teams.Core.json [hunk 33]
```diff
@@ -96,6 +96,66 @@
...
+          "Target": "callTranscript"
+        },
+        {
+          "Id": "09e85d0c-2551-40e7-a037-1f36fade0b0b",
+          "ApiChange": "Method",
+          "ChangedApiName": "GET",
+          "ChangeType": "Addition",
+          "Description": "Added the [get content](https://docs.microsoft.com/en-us/graph/api/callTranscript-get-content?view=graph-rest-beta) operation to the [callTranscript] resource.",
+          "Target": "callTranscript"
+        }
```
Comment (isvargasmsft) [inline changelog/Microsoft.Teams.Core.json:132]: Per my last comment, we can delete this changelog entry.
```suggestion
        }
```

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Let's move the information about getting the content of a call transcript to the get operation of the call transcript and delete the separate get-content topic.

File: changelog/Microsoft.Teams.Core.json (hunk 33 above)
Comment (mankadnandan) [reply changelog/Microsoft.Teams.Core.json]: Done

File: api-reference/beta/api/calltranscript-get-content.md (hunk 32 above)
Comment (mankadnandan) [reply api-reference/beta/api/calltranscript-get-content.md]: Sure. So I have removed the topic calltranscript-get-content.md and have updated calltranscript-get.md to include the content details, as per my understanding. Though I am not sure if it covers all the asks, so let me know if we need to change a bit more.

Also, the content page had some examples some more examples of showing how to use some other query params ($format), and headers? Shouldn't we include those examples too to make sure the api developer knows what all options they have while calling the /content endpoint?

File: api-reference/beta/api/calltranscript-get.md [hunk 34]
```diff
@@ -0,0 +1,127 @@
...
+doc_type: apiPageType
+---
+
+# Get callTranscript
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Retrieve a [callTranscript](../resources/calltranscript.md) object associated with an [onlineMeeting](../resources/onlinemeeting.md).
```
Comment (isvargasmsft) [inline api-reference/beta/api/calltranscript-get.md:16]: Please add a sentence explaining the difference between getting the transcript and getting the context of the transcript.

Comment (mankadnandan) [reply api-reference/beta/api/calltranscript-get.md:16]: Added

File: api-reference/beta/api/calltranscript-get-content.md (hunk 32 above)
Comment (mankadnandan) [reply api-reference/beta/api/calltranscript-get-content.md]: Included other examples as discussed.
